cli/internal/lockfile/yarn_lockfile.go

```diff
@@ -1,51 +1,29 @@
 package lockfile
 
 import (
-	"bufio"
 	"bytes"
 	"fmt"
 	"io"
-	"regexp"
-	"strings"
 
+	"github.com/andybalholm/crlf"
+	"github.com/iseki0/go-yarnlock"
 	"github.com/pkg/errors"
-	"gopkg.in/yaml.v3"
 )
 
-var rnLineEnding = regexp.MustCompile("\"|:\r\n$")
-var nLineEnding = regexp.MustCompile("\"|:\n$")
-var lineStart = regexp.MustCompile(`^[\w"]`)
-var double = regexp.MustCompile(`\:\"\:`)
-var quotedWhitespace = regexp.MustCompile(`\"\s\"`)
-
-// deals with colons
-// integrity sha-... -> integrity: sha-...
-// "@apollo/client" latest -> "@apollo/client": latest
-// "@apollo/client" "0.0.0" -> "@apollo/client": "0.0.0"
-// apollo-client "0.0.0" -> apollo-client: "0.0.0"
-var spaceDelimitedChars = regexp.MustCompile(`(\w|\")\s(\"|\w)`)
-
-// YarnLockfileEntry package information from yarn lockfile
-type YarnLockfileEntry struct {
-	// resolved version for the particular entry based on the provided semver revision
-	Version   string `yaml:"version"`
-	Resolved  string `yaml:"resolved"`
-	Integrity string `yaml:"integrity"`
-	// the list of unresolved modules and revisions (e.g. type-detect : ^4.0.0)
-	Dependencies map[string]string `yaml:"dependencies,omitempty"`
-	// the list of unresolved modules and revisions (e.g. type-detect : ^4.0.0)
-	OptionalDependencies map[string]string `yaml:"optionalDependencies,omitempty"`
-}
+var crlfLiteral = []byte("\r\n")
```

Comment:
We use the Uber Go style guide which says this:

```suggestion
var _crlfLiteral = []byte("\r\n")
```